lib/test/unit/testsuite.rb

```diff
@@ -76,9 +76,7 @@ def delete_tests(tests)
       # i.e. if the suite contains other suites, it counts the
       # tests within those suites, not the suites themselves.
       def size
-        total_size = 0
-        @tests.each { |test| total_size += test.size }
-        total_size
+        @tests.sum(&:size)
```

Comment:
We can't use this because this doesn't work on Ruby 2.1: https://github.com/test-unit/test-unit/actions/runs/8654971696/job/23735516900?pr=241#step:5:9

Reply:
Oh, I did not notice which versions were supported, sorry.
Not sure if I should close this PR, or just wait until this can be safely applied.

Reply:
Could you close this for now? We can revisit this later.